modules/cluster-logging-rn-5.4.11.adoc

```diff
@@ -0,0 +1,21 @@
+//module included in cluster-logging-release-notes.adoc
```

Comment:
```suggestion
// Module included in the following assemblies:
//
// * logging/cluster-logging-release-notes.adoc


```
**Suggestion**: Refer to the guideline and example [here](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata).